### PR TITLE
better jesus

### PR DESCRIPTION
**Pull request by e9x**
makes the client behave as if it were on a solid block instead of exiting the water

### CODE REVIEW

File: Horion/Module/Modules/Jesus.h [hunk 1]
```diff
@@ -5,11 +5,13 @@ class Jesus :
 {
 private:
 	bool wasInWater = false;
```
Comment (N0tExisting) [inline Horion/Module/Modules/Jesus.h]: You can remove this, as it is no longer used

**Comment by Intoprelised** [comment]
cum

**Comment by NRGJobro** [comment]
agred @Intoprelised

File: Horion/Module/Modules/Jesus.h (hunk 1 above)
Comment (e9x) [reply Horion/Module/Modules/Jesus.h:7]: I'm not sure what you mean. The jesus module is used in this repository.

**Pull request closed without merging**